scripts/performance/benchmark-cp

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+import argparse
+import subprocess
+import os
+
+from benchmark_utils import get_benchmark_script, get_summarize_script
+from benchmark_utils import human_readable_to_bytes, summarize, clean
+
+
+def benchmark_cp(args):
+    if not os.path.exists(args.result_dir):
+        os.makedirs(args.result_dir)
+    command = 'aws s3 cp %s %s' % (
+        args.source, args.destination)
+    
+    if args.recursive:
+        command += ' --recursive'
+
+    if args.quiet:
+        command += ' --quiet'
+    else:
+        print(command)
+
+    for i in range(args.num_iterations):
+        out_file = 'performance%s.csv' % i
+        out_file = os.path.join(args.result_dir, out_file)
+        benchmark_args = [
+            args.benchmark_script, command, '--output-file', out_file
+        ]
+        if not args.dry_run:
+            subprocess.check_call(benchmark_args)
+    
+    if not args.dry_run:
+        summarize(args.summarize_script, args.result_dir)
+
+        if not args.no_cleanup:
+            clean(args.destination, args.recursive)
```

Comment:
I think this should be wrapped in a `try/finally`.  I think we'd want to clean up even in the event that the bench mark scripts (or summarize scripts) fail.